mobile/lib/features/wallet/receive_screen.dart

```diff
@@ -78,112 +79,112 @@ class _ReceiveScreenState extends State<ReceiveScreen> {
+          padding: const EdgeInsets.all(20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              const SizedBox(height: 5),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  Expanded(
+                    child: OutlinedButton(
+                        onPressed: () => setState(() => _isLightning = true),
+                        style: OutlinedButton.styleFrom(
+                            minimumSize: const Size(20, 50),
+                            side: BorderSide(color: lightningColor),
+                            shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+                            backgroundColor: Colors.white),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            Text("Lightning",
+                                style: TextStyle(color: lightningColor, fontSize: 16)),
+                            Icon(Icons.bolt, color: lightningColor),
+                          ],
+                        )),
+                  ),
+                  const SizedBox(width: 10),
+                  Expanded(
                       child: OutlinedButton(
-                          onPressed: () => setState(() => _isLightning = true),
+                          onPressed: () => setState(() => _isLightning = false),
                           style: OutlinedButton.styleFrom(
-                              minimumSize: const Size(20, 50),
-                              side: BorderSide(color: lightningColor),
-                              shape:
-                                  RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
-                              backgroundColor: Colors.white),
+                            minimumSize: const Size(20, 50),
+                            side: BorderSide(color: bitcoinColor),
+                            backgroundColor: Colors.white,
+                            shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+                          ),
                           child: Row(
                             mainAxisAlignment: MainAxisAlignment.spaceBetween,
                             children: [
-                              Text("Lightning",
-                                  style: TextStyle(color: lightningColor, fontSize: 16)),
-                              Icon(Icons.bolt, color: lightningColor),
+                              Text("Bitcoin", style: TextStyle(color: bitcoinColor, fontSize: 16)),
+                              Icon(
+                                Icons.currency_bitcoin,
+                                color: bitcoinColor,
+                              ),
                             ],
-                          )),
```

Comment:
the diff here is a bit weird; did anything actually change here? did you swap the order of the buttons?

Reply:
It looks indeed weird, if the order has changed it was not intentional.